CPAC/func_preproc/func_preproc.py

```diff
@@ -1381,12 +1382,6 @@ def calc_motion_stats(wf, cfg, strat_pool, pipe_num, opt=None):
         name=f'gen_motion_stats_{pipe_num}',
         motion_correct_tool=motion_correct_tool)
 
-    # Special case where the workflow is not getting outputs from
-    # resource pool but is connected to functional datasource
```

Comment:
I am not sure about this.

Reply:
We should keep this comment in, though I cannot seem to trace what the original concern was. Maybe about directly connecting this on its own just to test it? I need to find ou t.

Reply:
It's not just the comment, it's the special case handling 
https://github.com/FCP-INDI/C-PAC/blob/e015640fefcc5f1b7e19a856585aaca88e3a542a/CPAC/func_preproc/func_preproc.py#L1315-L1322 https://github.com/FCP-INDI/C-PAC/blob/e015640fefcc5f1b7e19a856585aaca88e3a542a/CPAC/func_preproc/func_preproc.py#L1343-L1347 https://github.com/FCP-INDI/C-PAC/blob/e015640fefcc5f1b7e19a856585aaca88e3a542a/CPAC/generate_motion_statistics/generate_motion_statistics.py#L178-L186 that's [removed](https://github.com/FCP-INDI/C-PAC/commit/de7134775f9c4c245627abe245b4a5e84ec2b304) ↓ https://github.com/FCP-INDI/C-PAC/blob/92cb0774e7946c9364c0e0556008008c83eff7e0/CPAC/func_preproc/func_preproc.py#L1362-L1368 https://github.com/FCP-INDI/C-PAC/blob/92cb0774e7946c9364c0e0556008008c83eff7e0/CPAC/generate_motion_statistics/generate_motion_statistics.py#L171-L178 is why I ask.

To be fair, I don't see where `subject_id` was used in this special case before this PR, so it might be some legacy pre-1.8 thing